Content.Server/_Goobstation/Possession/PossessionSystem.cs

```diff
@@ -0,0 +1,279 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared._Goobstation.Devil;
+using Content.Shared._Goobstation.Possession;
+using Content.Server.Actions;
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Server.Stunnable;
+using Content.Shared.Administration.Logs;
+using Content.Shared.CombatMode.Pacification;
+using Content.Shared.Coordinates;
+using Content.Shared.Database;
+using Content.Shared.Examine;
+using Content.Shared.Ghost;
+using Content.Shared.Mind;
+using Content.Shared.Mindshield.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.Tag;
+using Content.Shared.Zombies;
+using Robust.Server.Containers;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Containers;
+using Robust.Shared.Map;
+using Robust.Shared.Spawners;
+using Robust.Shared.Timing;
+using Content.Server.Bible.Components;
+using Robust.Shared.Prototypes;
+using Content.Shared._Goobstation.Religion;
+
+namespace Content.Server._Goobstation.Possession;
+
+public sealed partial class PossessionSystem : EntitySystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly MobStateSystem _mobState = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly SharedMindSystem _mind = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly StunSystem _stun = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly ContainerSystem _container = default!;
+    [Dependency] private readonly ISharedAdminLogManager _admin = default!;
+    [Dependency] private readonly ActionsSystem _action = default!;
+    [Dependency] private readonly PolymorphSystem _polymorph = default!;
+    [Dependency] private readonly TagSystem _tag = default!;
+
+    private ProtoId<TagPrototype> _cannotSuicideAnyTag = "CannotSuicideAny"; // DeltaV - Don't use literals.
+    public override void Initialize()
```

Comment:
Formatting, modifiers, and naming
```suggestion
    private static readonly ProtoId<TagPrototype> CannotSuicideAnyTag = "CannotSuicideAny"; // DeltaV - Don't use literals.

    public override void Initialize()
```